internal/resources/grafana/contentTypeRoundTripper.go

```diff
@@ -0,0 +1,36 @@
+package grafana
```

Comment:
File name should be snake_case and I think that if it's only used in tests, then it could live in a _test.go file

Reply:
Ups!! Too much java during past week...